__tests__/src/getPropValue-babelparser-test.js

```diff
@@ -1079,4 +1079,45 @@ describe('getPropValue', () => {
       assert.equal(actual, expected);
     });
   });
+
+  describeIfNotBabylon('TSNonNullExpression', () => {
+    beforeEach(() => {
+      changePlugins((pls) => [...pls, 'typescript']);
+    });
+
+    it('should return string representation of a TSNonNullExpression of form `variable!`', () => {
+      const prop = extractProp('<div foo={bar!} />');
+      const expected = 'bar';
```

Comment:
shouldn't this be `bar!`, per the test description?
```suggestion
      const expected = 'bar!';
```